src/pages/create-index.html

```diff
@@ -1,16 +1,7 @@
-<link href="css/lib/angular-xeditable/xeditable.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet">
-
-<link href="css/lib/yasqe.min.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
-
-<link href="css/new-sparql.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
-
-<link href="css/graphs-config.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
```

Comment:
Why on earth this view had a dependency to graphs-config.css?! :(

Reply:
I think this is copy-pasted, when this functionality is implemented.